nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/api/Work.java

```diff
@@ -395,4 +395,14 @@ default boolean isIdempotent() {
         return true;
     }
 
+    /**
+     * Returns true if only the last scheduled work with a given {@link #getId()} should be executed. The WorkManager
+     * implementation can safely skip executions of coalesing work until the last with a given {@link #getId()}.
+     *
```

Comment:
This is right but not easy to understand the purpose, maybe something like:
When setting the coalescing flag to true you indicate to the work manager that if multiple works with the same id are scheduled you only care about the lastest execution. The goal is to skip useless executions to save resources, It is up to the work manager implementation to support or not this optimization.